roles/run_webpack/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+###
+# Run webpack to compile static assets (css and javascript).
+#
+# This role runs Webpack to compile and compress source javascript and styles
+# into css/js files. It depends on webpack and all its associated js dependencies
+# having been installed, usually through the 'build_npm' role.
+# 
+# The role expects that appropriate qa/prod-specific tasks will be defined in
+# package.json and callable via `npm run build:qa` and `npm run build:prod`.
+###
+---
+- name: webpack tasks
```

Comment:
Could this be slightly more descriptive. 'Run webpack build tasks.' Clearly it's provided up top too, but it's often handy to have a bit more descriptiveness in the actual play.